README.md

```diff
@@ -46,59 +58,66 @@ Running the tests locally requires a Postgres database for integration tests, an
-scripts/start
-
-# Stop the service and remove Docker volumes and networks created by the start script
-scripts/stop
+# Build images without using the Docker cache
+scripts/build --no-cache
 ```
 
-Any arguments provided to the build and start scripts are passed to the Docker Compose `build` and `up` commands, respectively. For example:
+## Run as an isolated service
+
+To test this service in isolation, use the provided scripts to start and stop a local instance. This relies on Docker Compose and will run direct dependencies, such as message queues and databases, as additional containers. Arguments given to the [`start`](./scripts/start) script will be passed through to the `docker-compose up` command.
 
 ```
-# Build without using the Docker cache
-scripts/build --no-cache
+# Start the service and attach to running containers (press `ctrl + c` to quit)
+scripts/start
 
 # Start the service without attaching to containers
 scripts/start --detach
+
+# Send a sample request to the /register endpoint
+curl -i --header "Content-Type: application/json" --request POST --data '{ "email": "test@email.com" }' http://localhost:3002/register
+
+# Stop the service and remove Docker volumes and networks created by the start script
+scripts/stop
 ```
 
-This service depends on an external Docker network named `ffc-demo` to communicate with other services running alongside it. The start script will automatically create the network if it doesn't exist and the stop script will remove the network if no other containers are using it.
+## Connect to sibling services
 
-The external network is declared in a secondary Docker Compose configuration (referenced by the above scripts) so that this service can be run in isolation without creating an external Docker network by using standard Docker Compose commands:
+To test this service in combination with other parts of the FFC demo application, it is necessary to connect each service to an external Docker network and shared dependencies, such as message queues. Start the shared dependencies from the [`ffc-demo-development`](https://github.com/DEFRA/ffc-demo-development) repository and then use the `connected-` [`scripts`](./scripts/) to start this service. Follow instructions in other repositories to connect each service to the shared dependencies and network.
 
 ```
-# Build containers
-docker-compose build
+# Start the service
+script/connected-start
 
-# Start the service is isolation
-docker-compose up
+# Stop the service
+script/connected-stop
 ```
 
-### Volume mounts on Windows Subsystem for Linux
-
-For the volume mounts to work correct via WSL it is necessary to either set up automounting of `/mnt/c`, or change the mount point for the shared drive from `/mnt/c` to `/c`. For a well-explained write-up, see [this blog post](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) by Nick Janetakis.
+## Deploy to Kubernetes

```

Comment:
I think it would be useful if the readme explained how the migrations work both locally and when deployed to Kubernetes.

Reply:
I've added some text around how the migrations work locally, why they don't run the check before every request, and how they're run under kubernetes (they're not, that's an outstanding ticket)